lib/Routing/Loader/AttributeRouteControllerLoader.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Routing\Loader;
+
+use ReflectionClass;
+use ReflectionMethod;
```

Comment:
Unused?

Reply:
it is used in line 30.

pimcore is using PHP-CS-Fixer's` global_namespace_import` rule.

Reply:
Sure, sorry :facepalm: